jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFMetadataProcessor.java

```diff
@@ -61,6 +86,11 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
                 JSFLogger.ROOT_LOGGER.lazyBeanValidationEnabled();
             }
         }
+        // Set a default buffer size as 1024 is too small
+        final JBossWebMetaData webMetaData = metaData.getMergedJBossWebMetaData();
+        if (!hasContextParam(webMetaData, "faces.FACELETS_BUFFER_SIZE")) {
```

Comment:
The param is "jakarta.faces.FACELETS_BUFFER_SIZE". Per the spec we should also check "facelets.BUFFER_SIZE" for compatibility.

Reply:
Looks like a type on my side, but it also says:

> The runtime must also consider the facelets.BUFFER_SIZE param name as an alias to this param name for backwards compatibility with existing facelets tag libraries.